generators/python/fastapi/versions.yml

```diff
@@ -1,6 +1,23 @@
 # yaml-language-server: $schema=../../../fern-versions-yml.schema.json
 # For unreleased changes, use unreleased.yml
 
+- version: 2.0.0
+  changelogEntry:
+    - type: break
```

Comment:
Can we also mention the removal of the patching code? And link to the FastAPI issue